openedx/core/djangoapps/programs/tests/test_utils.py

```diff
@@ -233,7 +235,6 @@ def test_get_program_for_certificates_id_not_exist(self):
         actual = utils.get_programs_for_credentials(self.user, credential_data)
         self.assertEqual(actual, [])
 
-    @httpretty.activate
```

Comment:
It's cleaner to apply the decorator once to the class instead of repeatedly to each method.